test/OperandCapabilities.cpp

```diff
@@ -73,135 +76,156 @@ TEST_P(EnumCapabilityTestV10, Sample) {
-    ::testing::ValuesIn(std::vector<EnumCapabilityCase>{
-        CASE1(EXECUTION_MODE, ExecutionModeSubgroupSize, SubgroupDispatch),
-        CASE1(EXECUTION_MODE, ExecutionModeSubgroupsPerWorkgroup,
-              SubgroupDispatch)}), );
+    ExecutionMode, EnumCapabilityTest,
+    Combine(
+        Values(SPV_ENV_UNIVERSAL_1_0, SPV_ENV_UNIVERSAL_1_1),
+        ValuesIn(std::vector<EnumCapabilityCase>{
+            CASE1(EXECUTION_MODE, ExecutionModeInvocations, Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeSpacingEqual, Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeSpacingFractionalEven,
+                  Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeSpacingFractionalOdd,
+                  Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeVertexOrderCw, Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeVertexOrderCcw, Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModePixelCenterInteger, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModeOriginUpperLeft, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModeOriginLowerLeft, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModeEarlyFragmentTests, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModePointMode, Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeXfb, TransformFeedback),
+            CASE1(EXECUTION_MODE, ExecutionModeDepthReplacing, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModeDepthGreater, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModeDepthLess, Shader),
+            CASE1(EXECUTION_MODE, ExecutionModeDepthUnchanged, Shader),
+            CASE0(EXECUTION_MODE, ExecutionModeLocalSize),
+            CASE1(EXECUTION_MODE, ExecutionModeLocalSizeHint, Kernel),
+            CASE1(EXECUTION_MODE, ExecutionModeInputPoints, Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeInputLines, Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeInputLinesAdjacency, Geometry),
+            CASE2(EXECUTION_MODE, ExecutionModeTriangles, Geometry,
+                  Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeInputTrianglesAdjacency,
+                  Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeQuads, Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeIsolines, Tessellation),
+            CASE2(EXECUTION_MODE, ExecutionModeOutputVertices, Geometry,
+                  Tessellation),
+            CASE1(EXECUTION_MODE, ExecutionModeOutputPoints, Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeOutputLineStrip, Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeOutputTriangleStrip, Geometry),
+            CASE1(EXECUTION_MODE, ExecutionModeVecTypeHint, Kernel),
+            CASE1(EXECUTION_MODE, ExecutionModeContractionOff, Kernel),
+        })), );
+
+INSTANTIATE_TEST_CASE_P(
+    ExecutionModeV11, EnumCapabilityTest,
+    Combine(Values(SPV_ENV_UNIVERSAL_1_1),
```

Comment:
Those tests have not yet been implemented.  This is a refactoring of existing tests.